src/web/package.json

```diff
@@ -10,6 +10,7 @@
     "@mui/styles": "5.6.1",
     "cra-template": "1.1.3",
     "formik": "2.2.9",
+    "pretty-quick": "^3.1.3",
```

Comment:
`npm install` in root folder, not to `npm install pretty-quick`.  Please revert this change

Sorry, haven't had time to implemented npm workspaces so that an install would install dependencies in every workspace